config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -6913,6 +6917,21 @@ spec:
                       type: string
                     description: Annotations defines the annotations placed on the envoy ingress controller deployment
                     type: object
+                  disruptionBudget:
```

Comment:
@ecojan can the pdb definition symmetric to existing https://github.com/banzaicloud/kafka-operator/blob/master/config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml#L6893-L6902 ?

Reply:
I reused the same structure but added a "strategy" field as well since Envoy pods are created as part of a deployment, hence a PDB can be used un "maxUnavailable" strategy.

Reply:
ok